include/crm/common/xml_internal.h

```diff
@@ -220,6 +220,7 @@ void pcmk__xe_remove_matching_attrs(xmlNode *element,
                                     void *user_data);
 int pcmk__xe_find_delete(xmlNode *xml, xmlNode *search);
 int pcmk__xe_find_replace(xmlNode *xml, xmlNode *replace);
+int pcmk__xe_find_update(xmlNode *xml, xmlNode *update);
```

Comment:
I'm thinking `pcmk__xe_{replace,update,delete}_match()`

Reply:
I avoided "match" because of https://github.com/ClusterLabs/pacemaker/pull/3386#discussion_r1517007391

I'm fine with it though

Reply:
meh, they're all questionable, but we can leave it how it is now

Reply:
"first_match()" might be better and isn't terribly long (although borderline), but it can wait